VERSION

```diff
@@ -1 +1 @@
-1.20.0-dev
+1.20.0
```

Comment:
i thought these changes happened on the release branch, rather than `main` - but not sure how its done

Reply:
per GOVERNANCE.md it's supposed to be with the release PR
the one sketchy bit is me setting the dates to the "wrong day" but given the list we have I'd be shocked if we cut the release today and I'm being optimistic about aiming for tomorrow :-P